internal/cli/refresher_opts.go

```diff
@@ -47,6 +51,14 @@ func (opts *RefresherOpts) RefreshAccessToken(ctx context.Context) error {
 	}
 	t, _, err := opts.flow.RefreshToken(ctx, config.RefreshToken())
 	if err != nil {
+		var target *atlas.ErrorResponse
+		if errors.As(err, &target) && target.ErrorCode == "INVALID_REFRESH_TOKEN" {
+			return fmt.Errorf(
+				"%w\n\nTo login, run: %s %s",
+				ErrInvalidRefreshToken,
+				config.BinName(),
+				"auth login")
+		}
```

Comment:
[Question/Not blocking] Is there anyway to get the command string ("auth login") from the cobra command definition. Just thinking if the command ever changes it would be nice if the text updated to reflect.

That said, changing command roots after releasing is a generally bad idea, so 🤷

Reply:
it gets tricky here but let me try on a follow up something I just think of